src/components/App/App.tsx

```diff
@@ -1,44 +1,30 @@
-import React, { useState, useEffect } from 'react';
+import React, {useState, useContext, useEffect} from 'react';
+import {QuestionContext, AllQuestionFormat} from '../../types'
+import {getQuestions} from '../../apiCalls'
 
-//types
-import { data } from '../../types';
 //components
-// import Home from '../Home/Home';
-// import Error from '../Error/Error';
-// import Journey from '../Journey/Journey';
-import Sourvey from '../Sourvey/Sourvey';
-import NavBar from '../NavBar/NavBar';
-// import GenerateReport from '../GenerateReport/GenerateReport';
+import Home from '../Home/Home';
+//import Error from '../Error/Error';
+//import Journey from '../Journey/Journey';
+import {Sourvey} from '../Sourvey/Sourvey';
+//import GenerateReport from '../GenerateReport/GenerateReport';
+//import './App.css';
 
-import './App.css';
+//create interface for context
 
-const Context = React.createContext({})
-
-
-const App = () => {
-
-  const [data, setData] = useState<data>();
-
-  const getQuestionsData = () => {
-
-    // setData();
-  }
+const App:React.FC = () =>{
+  const [questions, updateQuestions] = useState<AllQuestionFormat | null>(null);
+  useEffect(() => {
+    getQuestions().then((data) => updateQuestions(data))
+  }, []);
```

Comment:
Are you loading up the questions dynamically so that they have the same effect?

Reply:
useEffect is a bit misleading, this is essentially just a fetch for the data ( currently mocked on a temporary endpoint) that happens when the application is fired up. 

But yeah! This fetch will get all the questions and store them locally so we can display them to the DOM

Reply:
We'll be loading them up here through the API call, then displaying them with a map in another component.

Reply:
`useEffect` is just a built-in function in React that fires `onload`. This way will have the data available to us ready to display to the user.

 If your question is if `useEffect` adds a visual effect to each question, then I am afraid to tell that it is just the basic functionality to display data.